### PR TITLE
add profile picture upload

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

File: assembl/graphql/user.py [hunk 1]
```diff
@@ -74,6 +86,39 @@ def mutate(root, args, context, info):
         with cls.default_db.no_autoflush as db:
             user.username_p = args.get('username')
             user.real_name_p = args.get('name')
+
+            # add uploaded image as an attachment to the user
+            image = args.get('image')
+            if image is not None:
+                filename = os.path.basename(context.POST[image].filename)
```
Comment (ayyazdaniaryan) [inline assembl/graphql/user.py]: You should probably add CRUD logic check on the Attachment model.

File: assembl/models/attachment.py [hunk 2]
```diff
@@ -371,3 +371,40 @@ class ResourceAttachment(Attachment):
...
+            'profile_attachments',
+            cascade="all, delete-orphan"),
+    )
+    __mapper_args__ = {
+        'polymorphic_identity': 'agent_profile_attachment',
+        'with_polymorphic': '*'
+    }
+
+    crud_permissions = CrudPermissions(
+            P_READ, P_SYSADMIN, P_SYSADMIN, P_SYSADMIN,
```
Comment (ayyazdaniaryan) [inline assembl/models/attachment.py]: Particiapnts can Create, Sysadmin can RUD? Is that right?

Comment (vincentfretin) [reply assembl/models/attachment.py]: yes, that's right and only owner can update, remove

File: assembl/graphql/user.py (hunk 1 above)
Comment (vincentfretin) [reply assembl/graphql/user.py]: done in https://github.com/assembl/assembl/pull/265/commits/0e9e51aee2799a172f3663a53425cd6050974104

File: assembl/graphql/user.py [hunk 3]
```diff
@@ -45,12 +49,19 @@ def resolve_email(self, args, context, info):
         if include_emails or self.id == user_id:
             return self.get_preferred_email()
 
+    def resolve_image(self, args, context, info):
+        for attachment in self.profile_attachments:
+            if attachment.attachmentPurpose == 'PROFILE_PICTURE':
```
[the remark this answers is not in the document]
Comment (vincentfretin) [reply assembl/graphql/user.py]: done in https://github.com/assembl/assembl/pull/265/commits/54ab77f37bc2aa667b7f5d644a6037b639b24799

**Comment by vincentfretin** [comment]
@ayyazdaniaryan Do I have your go to merge?
Even if we don't implement the frontend now, I want this merged asap to avoid to resolve conflicts and change down revision of alembic step again.

**Comment by ayyazdaniaryan** [comment]
@vincentfretin 👍 for me. Go go go!

**Pull request merged**